website/static/js/addProjectPlugin.js

```diff
@@ -35,6 +34,9 @@ var AddProject = {
         self.newProjectDesc = m.prop('');
         self.newProjectCategory = m.prop(self.defaultCat);
         self.newProjectTemplate = m.prop('');
+        self.newProjectInstitutions = window.contextVars.currentUser.institutions.map(
```

Comment:
Can you avoid using `window.contextVars` easily? Pass institutions as a `param`?